plugins/views/frontend/public/javascripts/countly.models.js

```diff
@@ -1,9 +1,422 @@
 /*global CountlyHelpers, countlyCommon, $, countlySession, jQuery, countlyGlobal, Promise, CV, countlyVue*/
 
-(function() {
-    window.countlyViews = window.countlyViews || {};
-    CountlyHelpers.createMetricModel(window.countlyViews, {name: "views"}, jQuery);
-    var countlyViews = window.countlyViews;
+(function(countlyViews) {
+
+    CountlyHelpers.createMetricModel(countlyViews, {name: "views"}, jQuery);
+
+    countlyViews.service = {
+        fetchData: function(context) {
+            _segment = context.state.selectedSegment;
```

Comment:
Did this pass eslint?

I mean _segment is not defined above, it looks like a global var